data/remnant/remnant ships.txt

```diff
@@ -893,47 +893,32 @@ ship "Merganser"
 		licenses
 			Remnant
 		"cost" 6830000
-		"shields" 3000
-		"hull" 5000
+		"shields" 3500
+		"hull" 3000
 		"required crew" 1
 		"bunks" 3
 		"cargo space" 11
 		"fuel capacity" 600
-		"ramscoop" 1.75
-		"mass" 30
+		"ramscoop" 1.25
+		"mass" 90
```

Comment:
Because you modified the ship to be unable to support any additional weapons in its default configuration. That's a new change. The previous stats supported an extra Inhibitor, a Thrasher, or an EMP Torpedo Bay. You've set it to 0 net outfit space without providing any obvious removal outfits except for the engines.

I thought that was pretty clear from the comment I left.

Reply:
> Why does it need to be able to carry additional weapons in its current configuration?

So that it can do its job as an interceptor. Right now, it has the same armament as a Petrel with no room for more.